packages/web/docs/src/pages/get-started/schema-stitching.mdx

```diff
@@ -0,0 +1,467 @@
+
+To get started with Hive's CDN access, you'll need to create a CDN token from your target's
+**Settings** page. You'll see a separate section for managing and creating CDN tokens, called **CDN
+Access Token**.
+
+Click on **Create new CDN Token** to create a new CDN token. Describe your token with an alias, and
+click **Create**. Please store this access token securely. **You will not be able to see it again.**
+
+<Callout type="info">
+**Why are Registry and CDN tokens different?**
+
+We use a separate, externally managed storage to manage CDN tokens to ensure high availability of
+your schemas. This approach ensures that your GraphQL schemas are fully secured and highly
+available. CDN tokens are read-only and can only fetch schemas from the CDN. They do not have
+permissions to perform any other action on your Hive organization.
+
+</Callout>
+
+To use your access token, go to your target's page on Hive's dashboard and click on the **Connect to
+CDN** button. You will see a screen with instructions on how to obtain different types of artifacts
+from the CDN. For this guide, you can pick the **GraphQL SDL** artifact.
+
+<NextImage alt="CDN Token Form" src={cdnTokenImage} className="mt-10 max-w-2xl drop-shadow-md" />
+
+Copy the URL and let's try to fetch your schema using `curl` (replace `YOUR_HIVE_CDN_TOKEN` with
+your CDN token, and `CDN_ENDPOINT_HERE` with the endpoint you copied from Hive's dashboard):
+
+```bash
+curl -L -H "X-Hive-CDN-Key: YOUR_HIVE_CDN_TOKEN" CDN_ENDPOINT_HERE
+```
+
+You should see that Hive CDN returns your full GraphQL schema as an output for that command.
+
+### A simple gateway
+
+Now that you have your schema published, and the schema is available on Hive's CDN, you can use it
+to build a simple GraphQL gateway.
+
+In addition to the SDL endpoint exposed from Hive's CDN, you can also use the `/services` endpoint
+to fetch a list of all services and their metadata (url and name).
+
+As part of [Hive Client](/api-reference/client#javascript--nodejs-client) library, we are also
+exposing utilities for fetching and integrating your schema with your gateway.
+
+Here's a short snippet of how you can fetch the services information from Hive CDN, and start a
+GraphQL gateway using GraphQL-Yoga:
+
+```ts
+import { createServer } from 'node:http'
+import { buildSchema } from 'graphql'
+import { createYoga } from 'graphql-yoga'
+import { createServicesFetcher } from '@graphql-hive/client'
+import { stitchSchemas } from '@graphql-tools/stitch'
+import { stitchingDirectives } from '@graphql-tools/stitching-directives'
+
+const fetchServices = createServicesFetcher({
+  endpoint: process.env.HIVE_CDN_ENDPOINT,
+  key: process.env.HIVE_CDN_KEY
+})
+
+const { allStitchingDirectivesTypeDefs } = stitchingDirectives()
+
+async function main() {
+  const services = await fetchServices()
+  const subschemas = services.map(service => {
+    return {
+      schema: buildSchema(`${allStitchingDirectivesTypeDefs} ${service.sdl}`),
```

Comment:
```suggestion
      schema: buildSchema(service.sdl, { assumeValid: true, assumeValidSDL: true }),
```

Reply:
Stitching services are expected to have those already, but still `assumeValid` flags would be helpful. Stitching doesn't need them explicitly.